cluster/gce/gci/configure.sh

```diff
@@ -110,16 +110,33 @@ function download-or-bust {
   done
 }
 
+function is-preloaded {
+  local -r key=$1
+  local -r value=$2
+  if [[ -f ${KUBE_HOME}/preload_info ]] && (grep "${key}" "${KUBE_HOME}/preload_info" | grep "${value}" > /dev/null 2>&1);then
```

Comment:
By letting grep just error on a non-existent file, and using flags to make grep not output anything, this check could just be

```shell
grep -qs "${key},${value}" "${KUBE_HOME}/preload_info"
```